tests/crowdsourcing/test_analysis.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+from typing import Any, Dict, List
+from parlai.crowdsourcing.utils.analysis import AbstractResultsCompiler
+from collections import defaultdict
+
+try:
+    from mephisto.data_model.unit import Unit
+
+    class MockUnit(Unit):
+        pass
+
+
+except ModuleNotFoundError:
+
+    class MockUnit:
```

Comment:
Hmm if we can still define a `MockUnit` successfully here even if we can't import `mephisto`, what's the advantage to importing `mephisto` in the first place?

Reply:
It was just for data type check later. Although I agree that make the code look messy.

Reply:
Okay cool, makes sense :) Maybe you could add a comment saying that?